Java/JDI/jdi-cucumber-test-generator/src/main/webapp/WEB-INF/static/views/newSuits.html

```diff
@@ -127,6 +127,83 @@
 
                                 <div class="cucumber-test-editor-container" >
                                     <div id="steps_container"></div>
+                                    <div id="steps_container">
```

Comment:
Please, delete all added lines. It should look like this:
\<div class="cucumber-test-editor-container" >
    \<div id="steps_container"></div>
\</div>